tests/Application/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "babel-polyfill": "^6.26.0",
-    "chart.js": "^2.9.3",
+    "chart.js": "^3.7.3",
```

Comment:
And where do we have package.json files splited, one for Sylius 1.11 and second for Sylius 1.12?